egs/swbd/s5c/local/chain/tuning/run_tdnn_lstm_1e_gpudec.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# configs for 'chain' gpu decoding
+# NOTICE: 1. we need CUDA9.0 installed with correct driver version
+#         2. a GPU newer than K20
+#         3. as libkaldi-decoder.so needs cuda library, other libraries using
+#            libkaldi-decoder.so also needs it.
+
+stage=21
+
+echo "$0 $@"  # Print the command line for logging
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+set -x
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+dir=tmp/tdnn_lstm_1e_sp/
+model_dir=exp/chain/tdnn_lstm_1e_sp
+
+# model traning
+if [ $stage -le 19 ]; then
+bash local/chain/tuning/run_tdnn_lstm_1e.sh || exit 1;
+fi
+
+# acoustic inference
+if [ $stage -le 20 ]; then
+mkdir -p $dir/log
```

Comment:
Can you please indent this file appropriately?

Reply:
done